plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -299,10 +310,11 @@ def create_or_update_project(self, project_name, namespace, options):
             project = self.create_project(namespace, project_options)
 
             # add avatar to project
-            try:
-                project.avatar = open(options['avatar_path'], 'rb')
-            except IOError as e:
-                self._module.fail_json(msg='Cannot open {0}: {1}'.format(options['avatar_path'], e))
+            if options['avatar_path']:
```

Comment:
This change should not be part of this PR, but of a separate PR. (Bugfixes can be backported, new features cannot.)

Reply:
Here is the separate PR https://github.com/ansible-collections/community.general/pull/3927

It seems like the avatar_path is a recent feature addition and looks unreleased -- so back porting may not be necessary.